src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,10 +17,11 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.2.0" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.6.0" />
```

Comment:
Why is this package added? I think this is the package that requires Azure.Identity to be 1.7.0 or higher.

Reply:
Oh this was my mistake, I originally had a different approach in this PR but this isn't needed now. Thanks!